lib/src/kvs/sled/testing.rs

```diff
@@ -0,0 +1,307 @@
+#[cfg(test)]
+mod tests {
+
+	use crate::kvs::tx::Transaction;
+	use crate::kvs::{Datastore, Key, Val};
+	use std::fs;
+	use std::path::PathBuf;
+	use std::sync::atomic::{AtomicU16, Ordering};
+
+	/// This value is automatically incremented for each test
+	/// so that each test has a dedicated id
+	static TEST_ID: AtomicU16 = AtomicU16::new(1);
```

Comment:
This is still non-deterministic though right? We could just use string descriptors or hardcoded UUIDs stored in an enum.

Reply:
I use it mainly to be sure each instance starts with a different temporary directory. Technically the tests are still deterministic as they start with an empty directory. That said, I am not opposed to store them in a const or enum, but I am afraid it would become very large at some point.
What about using tempdir? https://crates.io/crates/tempdir